### DISCUSSION TITLE
8月8日付款到现在一直不能使用专业版，反映了，没人解决，解决不了请退款好吗

### DISCUSSION THREAD

**Discussion by juzhier** [getlantern/lantern-forum-cn#867] [Q&A]
付款邮箱leecx23184260@gmail.com，8月8日付款，一直收不到验证码，不能使用专业版，反馈过，都是让我发支付信息之后就消失了。我还专门发送邮件联系mike的邮箱，根本没有任何回复。我的支付凭证如下，要么你们把专业版的授权验证码发送到我的邮箱23184260@qq.com,要么直接升级我的蓝灯为专业版。要么就给我退款！
![微信图片_20240813083611](https://github.com/user-attachments/assets/bd9b3366-7c76-4a19-85db-2cd5464ce101)

**Comment by MaxMKMax** [comment]
@juzhier 您好. 非常抱歉. 我这里马上帮您提交问题解决. 请您等一等. 谢谢您的耐心等待.
您不要直接发邮箱给我. 我一般不出理由想的, 因为工作时间不够用. 被限制的.

**Comment by MaxMKMax** [comment]
@juzhier 
这是您的验证码来登入您的帐号:
106329
